### PR TITLE
FAQ component

### PR DESCRIPTION
**Pull request by neilZon**
**Description**
Add FAQ component 

closes: #37

### CODE REVIEW

File: libs/design-system/src/lib/Components/FAQ/FAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <div className="bg-transparent">
+      {faqList.map((faq, i) => {
+        const itemExpanded = expanded[i];
+        const isAboveExpandedItem = expanded.at(i + 1);
+        const isLast = i === expanded.length - 1;
+
+        return (
+          <div
+            onClick={() => toggleExpansion(i)}
+            className={`cursor-pointer py-7 md:px-7 ${
```
Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:37]: This needs some horizontal padding at small sizes too. May be helpful to wrap the story Template in a `p-4 bg-primary-gray-50` to see active white background

File: libs/design-system/src/lib/Components/FAQ/FAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  return (
+    <div className="bg-transparent">
+      {faqList.map((faq, i) => {
+        const itemExpanded = expanded[i];
+        const isAboveExpandedItem = expanded.at(i + 1);
+        const isLast = i === expanded.length - 1;
+
+        return (
+          <div
```
Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:35]: We could use https://reach.tech/accordion for accessibility, but feel free to leave a todo for the future. A quick improvement would be making this div a button

Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:35]: We could use https://reach.tech/accordion for accessibility. A quick improvement would be making this div a button

File: libs/design-system/src/lib/Components/FAQ/FAQ.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  );
+
+  const toggleExpansion = (index: number) => {
+    const newExpanded = [...expanded];
+    newExpanded[index] = !newExpanded[index];
+    setExpanded(newExpanded);
+  };
+
+  return (
+    <div className="bg-transparent">
```
Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:28]:
```suggestion
    <div>
```
```suggestion
    <div className="bg-transparent">
```

Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:28]:
```suggestion
    <div>
```
```suggestion
    <div className="bg-transparent">
```

File: libs/design-system/src/lib/Components/FAQ/FAQ.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+              <div
+                className={`${
+                  itemExpanded ? 'text-2xl font-semibold' : 'text-base'
+                } md:font-semibold ${variation === 'large' ? 'md:text-2xl' : ''}`}
+              >
+                {faq.question}
+              </div>
+              {itemExpanded ? <ChevronDown /> : <ChevronRight />}
+            </div>
+            {itemExpanded ? (
```
Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:59]: This could be `itemExpanded && <div....`

Comment (alse) [inline libs/design-system/src/lib/Components/FAQ/FAQ.tsx:59]: This could be `itemExpanded && <div....`

**Review by alse — CHANGES_REQUESTED** [review]
Looks good @neilZon , just left some small comments. Do you have prettier installed?

File: libs/design-system/src/lib/Components/FAQ/FAQ.tsx (hunk 3 above)
Comment (neilZon) [reply libs/design-system/src/lib/Components/FAQ/FAQ.tsx:28]: I think this comment may have duplicated a few times on the PR review, but just so I'm clear, the review is just to remove the `classname="bg-transparent"` from the div?

**Comment by alse** [comment]
@neilZon the changes look good, this just needs some dark mode styles

**Pull request merged**